app/src/hooks/useRoutes/useRoutes.tsx

```diff
@@ -53,6 +45,12 @@ export const routes: RouteMap = {
     map: "/i/map",
     data: "/i/data",
   },
+  app: {
+    dashboard: "/app/dashboard",
+    sports: "/app/sports",
+    bets: "/app/bets",
+    profile: "/app/profile",
+  },
```

Comment:
Since this is already the "app", I'd suggest the following: 

```suggestion
  dashboard: {
    latestTrends: () => `/`,
    sports: () => `/sports`,
    bets: () => `/bets`,
    profile: () => `/profile`,
  },
```

a function because we may pass query params to it.

Reply:
Got it 👍